spec/support/ansible_shared/automation_manager/refresh_configuartion_script_source.rb

```diff
@@ -14,27 +14,38 @@
   end
   let(:manager_class) { manager_class }
 
+  let(:tower_data) { Spec::Support::TowerDataHelper.tower_data }
+
+  let(:targeted_refresh_id) { tower_data['items']['targeted_refresh']['id'] }
+  let(:targeted_refresh_last_updated) { tower_data['items']['targeted_refresh']['last_updated'].utc }
+  let(:targeted_refresh_playbooks) { tower_data['items']['targeted_refresh']['playbooks'] }
+  let(:nonexistent_repo_id) { tower_data['items']['nonexistent_repo']['id'] }
```

Comment:
As the name suggests, a _nonexistent_repo_ is an invalid id. It was used to ensure that the refresher does not even try to load this repo when updating the _targeted_refresh_ one. When merging with the Rake task changes, it would be necessary to create such id. That can be achieved by creating some repo and then deleting it, making sure the id is captured and it neither belongs to any existing project in Tower, nor going to be reused unexpectedly in the future.